packages/purpose-template-process/src/services/readModelServiceSQL.ts

```diff
@@ -270,10 +278,11 @@ export function readModelServiceBuilderSQL({
     async getPurposeTemplateById(
       purposeTemplateId: PurposeTemplateId
     ): Promise<WithMetadata<PurposeTemplate> | undefined> {
-      return purposeTemplateReadModelServiceSQL.getPurposeTemplateById(
+      return await purposeTemplateReadModelServiceSQL.getPurposeTemplateById(
         purposeTemplateId
       );
     },
+    // TODO: remove if unused
     async getTenantById(id: TenantId): Promise<Tenant | undefined> {
       return (await tenantReadModelServiceSQL.getTenantById(id))?.data;
     },
```

Comment:
👍🏻  It should be leftover of previous scaffolding, could be handle in a separate PR.